scripts/release_notes.rb

```diff
@@ -7,25 +7,17 @@
 require 'uri'
 require 'net/http'
 require 'json'
-require 'pry-byebug'
 
 URL = 'https://projects.theforeman.org'
 
-# Check out installer modules
-# Compare packages in nightly vs release and retag dependencies
-
-# Create new release via API call
-# Create new search by open + release field
-
 unless @current_release_id = ARGV[0]
   puts "Usage: #{$0} RELEASE_ID"
   puts "Hint: 1.16.0 is 240"
   exit 1
 end
 
-# Move bugs with Release set to VERSION to NEXT-VERSION with status new
-def gather_issues(offset=0)
-  url = "#{URL}/projects/foreman/issues.json?status_id=closed&offset=#{offset}&limit=100&release_id=#{@current_release_id}"
+def gather_issues(offset = 0, feature = true)
+  url = "#{URL}/projects/foreman/issues.json?status_id=closed&offset=#{offset}&limit=100&f[]=cf_12&op[cf_12]==&v[cf_12][]=#{@current_release_id}&f[]=tracker_id&op[tracker_id]=#{feature ? '=':'!'}&v[tracker_id][]=2"
```

Comment:
Right now you also have to loop over the projects because we no longer inherit versions.

Reply:
Right, i'll fix this to handle multiple versions.